src/ray/gcs/gcs_server/store_client_kv.cc

```diff
@@ -61,11 +63,12 @@ void StoreClientInternalKV::Get(
   RAY_CHECK_OK(delegate_->AsyncGet(
       table_name_,
       MakeKey(ns, key),
-      [callback = std::move(callback)](auto status, auto result) {
-        RAY_CHECK(status.ok()) << "Fails to get key from storage " << status;
+      {[callback = std::move(callback)](auto status, auto result) {
+         RAY_CHECK(status.ok()) << "Fails to get key from storage " << status;
         callback(result.has_value() ? std::optional<std::string>(result.value())
-                                    : std::optional<std::string>());
-      }));
+                                     : std::optional<std::string>());
+       },
+       io_context_}));
```

Comment:
Shouldn't `StoreClientInternalKV::xxx` accepts postable?

Reply:
we will do it in another PR. `StoreClientInternalKV::xxx`'s caller will be the end caller, i.e. the Managers, so let's make it step 2.

Reply:
kk, sg